src/devices/README.md

```diff
@@ -251,6 +255,138 @@ iot/
           Foo.Tests.cs
 ```
 
+When creating a new binding, please follow the recommendations [from corefx coding guideliness](https://github.com/dotnet/corefx/tree/master/Documentation#coding-guidelines). On top, we recommend to use very descriptive namings, self explanatory names. Please make sure you don't use abbreviation. Describe any unit a sensor may use as well as the usual range of values. Check as well [the devices convention](../../Documentation/Devices-conventions.md). You will find below couple of examples of do and don't.
```

Comment:
you might want to update that with runtime coding guideliness equivalent somewhere from here:
https://github.com/dotnet/runtime/tree/master/docs/coding-guidelines